File: Easy/146/polygonPerimeter.c
#include <stdio.h>
#include <math.h>

int main()
{
    int numSides;
    float circumradius; 
    printf("Enter number of sides: ");
    scanf("%d", &numSides);
    printf("Enter circumradius size: ");
    scanf("%f", &circumradius);

    float oneSide, perimeter;

    oneSide = circumradius * (2 * sin(M_PI/numSides));
    perimeter = oneSide * numSides;
    printf("%6.3f\n", perimeter);
}

File: Easy/146/polygonPerimeter.py
#!/usr/bin/python

import sys,math

def main(argv):
    numSides = int(argv[0])
    circumradius = float(argv[1])

    oneSide = circumradius*(2*math.sin(math.pi/numSides))
    print str(round(oneSide*numSides, 3))
    if __name__ == "__main__":
        main(sys.argv[1:])
